ProcessLib/HydroMechanics/HydroMechanicsFEM.h

```diff
@@ -413,6 +413,41 @@ class HydroMechanicsLocalAssembler : public LocalAssemblerInterface
         std::vector<double>& local_b_data, std::vector<double>& local_Jac_data,
         LocalCoupledSolutions const& local_coupled_solutions);
 
+    double const getFluidDensity(
```

Comment:
Please move these two functions to HydroMechanicsProcessData.h, and add a comment such that 'It will be removed after the fluid properties are replaced with MPL'